install/image-references

```diff
@@ -18,3 +19,9 @@ spec:
     from:
       kind: DockerImage
       name: docker.io/openshift/origin-machine-config-daemon:v4.0.0
+  # This one is special, it's the OS payload
+  # https://github.com/openshift/machine-config-operator/issues/183
+  - name: machine-os-content
```

Comment:
Do we still actually need this now? We're just using the configmap, right?

Reply:
The `image-references` file is used by `oc adm release new` (and hence the release operator) to define which images are substituted in the payload.

> (And then teach CI to inject the real value there when assembling the release payload).

This happens today.  The only thing that needed special intervention was Clayton created the `machine-os-content` imagestream.

Reply:
Gotcha. I'm still confused as to how the pullspec goes from imagestream to the configmap? E.g. in this PR, I see no other reference to `machine-os-content` so I'm guessing it's happening elsewhere?

Reply:
To answer my own question, the image references are used in a blanket find & replace throughout all the other manifests. There's nothing else that needs to directly say "replace this configmap value with the one from the imagestream".